packages/tokens/package.json

```diff
@@ -9,6 +9,7 @@
     "@semantic-release/exec": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
     "@synergy-design-system/eslint-config-syn": "workspace:*",
+    "@synergy-design-system/stylelint-config-syn": "workspace:*",
```

Comment:
I just adjusted the pattern to use `*` instead of `^`, because this is what we are currently using on all other projects. Otherwise, this seems fine and ready to merge from my point of view 👍